pincer/utils/insertion.py

```diff
@@ -26,6 +26,29 @@ def should_pass_cls(call: Union[Coro, Callable[..., Any]]) -> bool:
     args = getfullargspec(call).args
     return len(args) >= 1 and args[0] in ["self", "cls"]
 
+
+def should_pass_gateway(call: Union[Coro, Callable[..., Any]]) -> bool:
+    # TODO: fix docs
+    """
+    Checks whether a callable requires a dispatcher as last parameter.
+
+    Parameters
+    ----------
+    call: Union[Coro, Callable[..., Any]]
+        The callable to check.
+
+    Returns
+    -------
+    bool
```

Comment:
I don't know if it's required but shouldn't it be `` :class:`bool` `` or sth like that?

Reply:
Its not required I think